setup.py

```diff
@@ -42,17 +42,16 @@
                                 'schemas/v1.1.0-dev1/salad/schema_salad/metaschema/*.md',
                                 'cwlNodeEngine.js']},
       install_requires=[
-          'setuptools >= 18.5',  # needed by html5lib
-          'requests',
+          'setuptools',
+          'requests>=1.0',
           'ruamel.yaml == 0.12.4',
-          'rdflib >= 4.1.0',
-          'rdflib-jsonld >= 0.3.0',
+          'rdflib >= 4.2.0, < 4.3.0',
+          'rdflib-jsonld == 0.3.0',
+          'html5lib >=0.90, <= 0.9999999',
```

Comment:
This is what planemo uses - I think it is the better approach if it is attempting to solve the same problem with setuptools.

```
html5lib>=0.9999999,!=0.99999999,!=0.999999999,!=1.0b10
```